Added Russian localization file and fixed Cyrillic symbols in the font

Sidiusz opened the pull request:
1. I fixed the Cyrillic symbols, now it does not replace other symbols, but uses its own symbol slots (Still doesn't work when writing text in Russian in chat)
2. Added Russian localization file

scomans commented:
You also have to add an entry for russian otherwise it will not be listed in the dropdown: https://github.com/tiltedphoques/TiltedEvolution/blob/dev/Code/skyrim_ui/src/app/transloco-root.module.ts#L28
I added the german entry with the german word for german so you might also use the russian word for russian.

Dragonisser commented:
Why are you linking away from skyrim-together.com?

Sidiusz commented:
> You also have to add an entry for russian in the file below. Otherwise it will not be listed in the dropdown: https://github.com/tiltedphoques/TiltedEvolution/blob/dev/Code/skyrim_ui/src/app/transloco-root.module.ts#L28 I added the german entry with the german word for german so you might also use the russian word for russian.

Added, thanks!

Sturlen commented on Code/skyrim_ui/src/app/transloco-root.module.ts:
```diff
@@ -23,12 +23,18 @@ export class TranslocoHttpLoader implements TranslocoLoader {
+        availableLangs: [
+          { id: 'en', label: 'English' },
+          { id: 'de', label: 'Deutsch' },
+		  { id: 'ru', label: 'Русский' },
```
This should be indented so it it lines up with the other languages in the list.

Dragonisser reviewed (changes requested):
Why are you linking away from skyrim-together.com?

Sidiusz commented:
Our VK community has all the links to the project (website, github, wiki)
I replaced the link because the community is in Russian and Russian-speaking people will be available in a language they understand, and if they want details and the original text, all the links are there

Sidiusz commented on Code/skyrim_ui/src/assets/i18n/ru.json:
```diff
@@ -0,0 +1,156 @@
+      "LAUNCH_PARTY": "Создать группу",
+      "INVITES": "Приглашения",
+      "NO_INVITES": "Вас никто не пригласил.",
+      "EXPERIENCE_WARNING": "Для получения наилучшего опыта от прохождения заданий, следуйте нашим рекомендациям, которые вы можете найти по ссылке: <b>vk.com/skyrim_multiplayer</b>. Вот краткий список основных рекомендаций:",
```
Our VK community has all the links to the project (website, github, wiki)
I replaced the link because the community is in Russian and Russian-speaking people will be available in a language they understand, and if they want details and the original text, all the links are there

yuuklahoma replied:
> 

Add both.

RobbeBryssinck replied:
We don't control those sources, and are thus unofficial. Point it to the official website.

Sidiusz replied:
Added, sorry

RobbeBryssinck replied:
Adding both is fine too, but it still doesn't explicitly mention that the vk link is from an unofficial source. That needs to be mentioned.

Sidiusz replied:
> Adding both is fine too, but it still doesn't explicitly mention that the vk link is from an unofficial source. That needs to be mentioned.

I do not think it is necessary because VK is a Russian social network (like facebook) and for the users of this network there are no safety risks

RobbeBryssinck replied:
No, it is necessary, since we, the maintainers, don't control that site. In fact, it's better if it's removed. Russian users can very easily use Google translate on our site, and find the vk board through other means. Resolve the merge conflicts, remove the vk link, and it can be merged.

Sidiusz commented:
![Screenshot_1](https://user-images.githubusercontent.com/20544939/181689304-1c8deadf-a10c-46be-94a9-af0334cff919.png)

scomans commented on ru.json:
```diff
@@ -0,0 +1,155 @@
+{
```
file is in the wrong folder or just a duplicate
